packages/@groupby/elements-gb-tracker-plugin/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@groupby/elements-gb-tracker-plugin",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/index.js",
```

Comment:
This is missing the `types` and `esnext` keys. Follow the order found in the other plugins.

Reply:
I got it from *somewhere*...That implies that the `types` and `esnext` keys are missing elsewhere, or that they were missing and have been fixed.

Reply:
Looks like it's the SAYT plugin, and apparently we didn't fix it before releasing it.